packages/node-client/lib/index.ts

```diff
@@ -733,6 +734,30 @@ export class Nango {
         return response.data;
     }
 
+    /**
+     * Wait for connection to be created
+     * @param integrationKey - The integration key
+     * @param userId - The user ID
+     * @returns A promise that resolves when the connection is created
+     */
+    public async waitForConnection(integrationKey: string, userId: string): Promise<ApiPublicConnection> {
+        const maxAttempts = 30;
+        const delayBetweenAttemptsMs = 2000;
```

Comment:
we could have longer steps, especially the initial one. Going through an auth flow isn't very quick

Reply:
Updated